components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -47,4 +47,28 @@ class ActionSheetTest: XCTestCase {
     // Then
     XCTAssertEqual(actionSheet.title, "New title")
   }
+
+  func testAccessibilityIdentifiers() {
+    // Given
+    let testIdentifierOne = "Test One"
+    let testIdentifierTwo = "Test Two"
+    let actionSheet = MDCActionSheetController(title: nil)
+
+    // When
+    let actionOne = MDCActionSheetAction(title: "Title", image: nil, handler: nil)
+    actionOne.accessibilityIdentifier = testIdentifierOne
+    actionSheet.addAction(actionOne)
+
+    let actionTwo = MDCActionSheetAction(title: "Title", image: nil, handler: nil)
+    actionTwo.accessibilityIdentifier = testIdentifierTwo
+    actionSheet.addAction(actionTwo)
+
+    // Then
+    if let first = actionSheet.actions.first {
+      XCTAssertEqual(testIdentifierOne, first.accessibilityIdentifier)
+    }
+    if let last = actionSheet.actions.last {
+      XCTAssertEqual(testIdentifierTwo, last.accessibilityIdentifier)
+    }
+  }
```

Comment:
Is there any way for this test to verify that the accessibility identifier was set on the action sheet view?